clang/test/Driver/freebsd.c

```diff
@@ -191,11 +191,6 @@
 // RUN:   | FileCheck -check-prefix=CHECK-MIPS64-CPU %s
 // CHECK-MIPS64-CPU: "-target-cpu" "mips3"
 
-// Check that the integrated assembler is enabled for SPARC64
```

Comment:
why is this part here (before sparc64 is removed from FreeBSD)? It is not covered by the commit message anyway, afaict

Reply:
I already removed the other OS specific tests as I enabled IAS across the board awhile ago.

Reply:
Maybe worth noting that in the commit message too but either way LGTM for FreeBSD

Reply:
Done.